test/gtest/ucs/arch/test_x86_64.cc

```diff
@@ -63,6 +63,58 @@ class test_arch : public ucs::test {
     out:
         return result;
     }
+
+#ifdef __AVX__
+    void nt_buffer_transfer_test(ucs_arch_memcpy_hint_t hint) {
```

Comment:
can we make it part of 
UCS_TEST_SKIP_COND_F(test_arch, memcpy

Reply:
I was not able to make it compile with 'UCS_TEST_SKIP_COND_F'

what about replacing nt_buffer_transfer entry function with memcpy(). I have attached a patch, please check
[gtest_fix.txt](https://github.com/openucx/ucx/files/15165975/gtest_fix.txt)

Reply:
ok, let's keep as is

Reply:
Keeping the test code as currently in test/gtest/ucs/arch/test_x86_64.cc